unit_tests/authorize.test.js

```diff
@@ -0,0 +1,38 @@
+const assert = require("assert");
+const sinon = require("sinon");
+const userControler = require("../api/users/users.controller");
+const userModel = require("../api/users/users.module");
+const should = require("should");
+
+describe("unit test", () => {
+  describe("authorization", () => {
+    let sandbox;
+    let findByIdStub;
+    let actualResult;
+    const jwtToken =
+      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjVmMmJjMmEwNzA2YjdmM2U2OWNhMDRmNSIsImlhdCI6MTU5NjcwNDcxN30.XIK7OCtNZZ3A_LIJvDrfqiEazQG8V6Iae1Pd-96LvSk";
```

Comment:
remove this token

you should login 
after that receive token and store to variable